opentelemetry/proto/profiles/v1/alternatives/normalized/normalized.proto

```diff
@@ -0,0 +1,141 @@
+syntax = "proto3";
+
+package opentelemetry.proto.profiles.v1.alternatives.normalized;
+
+import "opentelemetry/proto/common/v1/common.proto";
+import "opentelemetry/proto/resource/v1/resource.proto";
+
+option go_package = "go.opentelemetry.io/proto/otlp/profiles/v1/normalized";
+
+
+// A pointer from a profile to a trace span.
+message Link {
+  // A unique identifier of a trace that this linked span is part of. The ID is a
+  // 16-byte array.
+  bytes trace_id = 1;
+  // A unique identifier for the linked span. The ID is an 8-byte array.
+  bytes span_id = 2;
+}
+
+message AttributeSet {
+  repeated opentelemetry.proto.common.v1.KeyValue attributes = 1;
+  uint32 dropped_attributes_count = 2;
+}
+
+message Sample {
+  repeated uint64 location_ids = 1;
```

Comment:
I wonder if those should be int32/uint32 I don't think we'll overflow this specially since they refer slice/array indexes.

This is not super important for the wire format since proto will varint encode it but for the collector this will allow it to buffer more as the memory footprint will be lower per profiles.